xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -1306,10 +1560,23 @@ This check is conceptually similar to `(void*)(&left) < (void*)(&right)`.
       </TypeParameters>
       <Parameters />
       <Docs>
-        <typeparam name="T">The type of object whose size is retrieved.</typeparam>
-        <summary>Returns the size of an object of the given type parameter.</summary>
-        <returns>The size of an object of type <typeparamref name="T" />.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="T">The type whose size is retrieved.</typeparam>
+        <summary>Returns the size of a value of the given type parameter.</summary>
+        <returns>The size (in bytes) of a value of type <typeparamref name="T" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+> [!CAUTION]
+> This API returns the size of the _managed_ view of the type. For the size of the _unmanaged_ view of the type, such as for interop purposes, use <xref:System.Runtime.InteropServices.Marshal.SizeOf%2A>.
+
+This API corresponds to the `sizeof` opcode.
+
+If `T` is a reference type, the return value is the size of the reference itself, or `IntPtr.Size`. See ECMA-335, Sec. Sec. III.4.25  ("sizeof - load the size, in bytes, of a type") for more information.
```

Comment:
`, or IntPtr.Size` is going to read as if its one or the other to some people.

It might be better to just specify `sizeof(nint)` or `sizeof(void*)`